src/decays/specializations/H/decay_H_to_lbarl.inc

```diff
@@ -41,5 +41,22 @@ double CLASSNAME::get_partial_width<Higgs, bar<ChargedLepton>::type, ChargedLept
       res_P *= 1. + get_alpha(context)/Pi*17./4.;
    }
 
+   if (static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_higgstools)) ||
+       static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_lilith)) ||
+       static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::calculate_normalized_effc))
```

Comment:
Would you mind adapting these if-statements as
~~~.cpp
if (flexibledecay_settings.get(FlexibleDecay_settings::calculate_normalized_effc) != 0)
~~~